tests/ContinuousRandomization_TEST.cxx

```diff
@@ -27,13 +27,24 @@ ParticleDef getParticleDef(const std::string& name)
 
 TEST(ContinuousRandomization, Constructor)
 {
-    auto p_def = MuMinusDef();
+    auto p_def = EPlusDef();
```

Comment:
Was there a reason to change the particle?

Reply:
Because EPlusDef, per default, includes an only-stochastic cross section (annihilation), while muons do not.

This is basically a check for [this line](https://github.com/tudo-astroparticlephysics/PROPOSAL/pull/319/files#diff-751770671f6e83fb8810b1b206768de58df5361aa839a3120e2b8888bd256a5aR18) in the code.